src/lucky/redirectable.cr

```diff
@@ -24,6 +24,55 @@
 # method that takes a `String`. However, it's recommended you pass a
 # `Lucky::Action` class if possible because it guarantees runtime safety.
 module Lucky::Redirectable
+  # Redirect back with a `Lucky::Action` fallback
+  #
+  # ```crystal
+  # redirect_back fallback: Users::Index
+  # ```
+  def redirect_back(fallback : Lucky::Action.class, status = 302)
+    redirect_back fallback.route, status
+  end
```

Comment:
```suggestion
  def redirect_back(*, fallback : Lucky::Action.class, status = 302)
    redirect_back fallback.route, status
  end
```

This forces the user to use a name arg. I think this is good since the args are not clear if the end-user leaves off the key.

This would need to be added in the other versions as well.

Reply:
oh, that's sneaky.... I like it!